locust/test_object.py

```diff
@@ -0,0 +1,193 @@
+
+import time
+
+class TestSuite(object):
+    def __init__(self, **kwargs):
+        self._id = 'TSU-'+int(time.time())
+        self._name = kwargs.get('name', None)
+        self._test_cases = kwargs.get('test_cases', [])
+        self._path = kwargs.get('path', None)
+
+    @property
+    def unique_id(self):
+        return self._id
+    
+    @unique_id.setter
+    def unique_id(self, value):
+        self._id = value
+
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, value):
+        self._name = value
+
+    @property
+    def get_test_cases(self):
+        return self._test_cases
+
+    @property
+    def append_test_case(self, test_case):
```

Comment:
it should not be decorated as property
`append_test_case` should be rename as `set_test_case` since if we use dict we can replace or add new key

Reply:
the first step before we set the new test_case to test_cases dict, we set the test_suite_id of this new test_case with self._id